hw/ip/aes/dv/aes_base_sim_cfg.hjson

```diff
@@ -39,6 +39,8 @@
   // Add additional tops for simulation.
   sim_tops: [ "aes_bind", "aes_cov_bind"]
 
+    xcelium_cov_refine_files: ["{proj_root}/hw/ip/aes/dv/cov/refines/aes_common_assert.vRefine", "{proj_root}/hw/ip/aes/dv/cov/refines/aes_remove_for_v2s.vRefine"  ]
```

Comment:
is `aes_remove_for_v2s` temporary? If so, please add TODO to remove it later.

Reply:
done!